web/templates/index.html

```diff
@@ -16,5 +16,5 @@
     */ -}}
 {{ $Prefix := .Prefix }}
 {{ range .Tickets }}
-<a href="{{$Prefix}}/Ticket/Display.html?id={{ .ID }}">{{ .ID }}</a>: {{ .Subject }} (<i>{{.Status}}</i>)<br>
+XX <a href="{{$Prefix}}/Ticket/Display.html?id={{ .ID }}">{{ .ID }}</a>: {{ .Subject }} (<i>{{.Status}}</i>)<br>
```

Comment:
XX ?

Reply:
leftover, nice catch

Reply:
not really sure where this templated is used by the way